doc/manual/develop.rst

```diff
@@ -94,6 +94,24 @@ run the ``*clean`` targets: otherwise e.g. ``paths.mk`` will
 be deleted before subdirectory ``*clean`` targets are called that
 depend on information in it.
 
+Debugging
+---------
+To enable debugging in the web interface, make sure you have set
```

Comment:
Line 67 also contains this information, although it doesn't hurt to also put it here.

Reply:
Thanks, I've moved the two paragraphs there to sections where I think they are more appropriate.